ruzzle_bare_minimum.py

```diff
@@ -124,31 +140,51 @@ def write_words_to_file(words_info, print_info=False):
             print('Number of words:', len(high_scores))
             print('Total score:', sum(info[0] for word, info in high_scores))
 
+
 def get_letter_pts(l):
     """Return point value for each letter (no bonuses)"""
-    letter_points = {'A': 1,'B': 4,'C': 4,'D': 2,'E': 1,'F': 4,'G': 3,'H': 4,'I': 1, 'J': 10,'K': 5,'L': 1,'M': 3,'N': 1,'O': 1,'P': 4,'Q': 10,'R': 1,'S': 1,'T': 1,'U': 2,'V': 4,'W': 4,'X': 8,'Y': 4,'Z': 8}
+    letter_points = {'A': 1, 'B': 4, 'C': 4, 'D': 2, 'E': 1, 'F': 4, 'G': 3, 'H': 4, 'I': 1, 'J': 10, 'K': 5, 'L': 1,
+                     'M': 3, 'N': 1, 'O': 1, 'P': 4, 'Q': 10, 'R': 1, 'S': 1, 'T': 1, 'U': 2, 'V': 4, 'W': 4, 'X': 8,
+                     'Y': 4, 'Z': 8}
     return letter_points[l]
 
+
 def get_prefixes():
     """returns a list of lists of prefixes of a certain number of letters"""
     prefixes = []
     for i in range(2, 7):
-        with open(main_dir/f'prefixes{i}L.txt') as file:
+        with open(main_dir / f'prefixes{i}L.txt') as file:
             prefixes.append(set(file.read().splitlines()))
     return prefixes
 
+
+def word_mults_to_int_array():
+    int_word_mults = []
+    for i in range(4):
+        int_word_mults.append([None] * 4)
+        for j in range(4):
+            int_word_mults[i][j] = int(word_mults[i][j]) if word_mults[i][j] in '23' else 1
```

Comment:
Would it be better to pass word_mults as a parameter rather than using it globally? Also, would initializing int_word_mults to have 1 instead of None be better, since 1 is the default? This would get rid of the single line if else assignment line.

Reply:
You would still need to test for values in word_mults, so it doesn't make a significant difference.